### ISSUE TITLE
PriemVue with Vite 5: Missing common Css Styles

### ISSUE THREAD

**Issue by cuberinooo** [primefaces/primevue#5249]
### Describe the bug

Im using the latest PrimeVue Version 3.48.0 and I just upgraded Vite from 4 to 5. After migration the common styles of each primevue components are not loaded anymore.  I saw that they have some breaking changes: See https://vitejs.dev/guide/migration.html. 

E.g. On the screenshot you can see some styles of the multiselect component which are not loaded anymore when I upgrade vite to 5.
![Screenshot from 2024-02-09 09-15-18](https://github.com/primefaces/primevue/assets/26199515/eeb4a952-dc3c-4326-b084-1d08e62d51d2)

Btw. I use Prestige Template and all styles of the template are loaded properly. These are included on the main.ts file.

Has anybody faced the same issue? Is there are workaround to fix this ? Im not even sure if this problem is related to primevue. Since only the upgrade from vite changes the behaviour. But I want to make sure if I have to change the import of the styles. Since Primevue did also some changes on styles. E.g. the primevue.min.css got removed and also they added @layer. 
(See https://github.com/primefaces/primevue/issues/4647 or: https://github.com/primefaces/primevue/issues/4110)

Maybe that`s useful to know: 
I use TS and everything is configured to ESM. We did not import a theme. As I said only the prestige Templated were used and imported to main.ts and the other styles were coming directly from primevue components.

### Reproducer

-

### PrimeVue version

3.48.0

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vite

### Steps to reproduce the behavior

Upgrade to vite 5 from 4 and see that the common styles of each primevue components are not loaded anymore. 

### Expected behavior

Common Styles of each primevue component should loaded

**Comment by mertsincan** [comment]
Interesting! We'll check and get back to you. I set a milestone for it.

**Comment by marknn3** [comment]
I wanted to create a new issue, but it looks like this issue is related/similar:
Since PrimeVue 3.48.0 styles are missing in the html head. This seems unrelated to vite4->5 migration as stated by the OP.
With PrimeVue v3.47.2:
![2024-02-11 22_34_16-PrimeVue missing styles since 3 48 0-1](https://github.com/primefaces/primevue/assets/4366965/f6c84786-cf23-4ad5-92fe-45449f5b6aa4)
With PrimeVue v3.48.0 (No other changes):
![2024-02-11 22_33_21-PrimeVue missing styles since 3 48 0-2](https://github.com/primefaces/primevue/assets/4366965/4b19bd38-4104-41df-a8d5-b1b77b85359e)

Minimum source to reproduce the issue:
[primevue-style-issue-3.48.0.zip](https://github.com/primefaces/primevue/files/14234150/primevue-style-issue-3.48.0.zip)

**Comment by marknn3** [comment]
Found the issue!
Since PrimeVue 3.48.0 styles are being moved to theme files. This does require an update of the theme files bundled with an existing project.
This is technically a breaking change, because is does require a manual copy of the theme file(s) to upgrade.

**Comment by eneeio-nm** [comment]
I too am surprised that this change wasn't flagged as a major release. It's going to take a couple of additional days dev time to upgrade our web platform to utilise the latest PrimeVue release (3.48.x).

**Comment by eneeio-nm** [comment]
> I wanted to create a new issue, but it looks like this issue is related/similar: Since PrimeVue 3.48.0 styles are missing in the html head. This seems unrelated to vite4->5 migration as stated by the OP. 

I agree - this doesn't have anything to do with upgrade Vite from 4 to 5.

**Comment by tugcekucukoglu** [comment]
For more information please check [#5214 ](https://github.com/primefaces/primevue/issues/5214)

**Comment by vzakharov** [comment]
Can someone please clarify how to fix this in an existing project? We're currently importing as
```
import './assets/our-theme.css'
import 'primeflex/primeflex.css'
import 'primevue/resources/primevue.min.css'
import 'primeicons/primeicons.css';
```

Do I have to now include manual imports for every component, or is there a more automated way?
